spec/features/user_updates_book_spec.rb

```diff
@@ -0,0 +1,53 @@
+require 'rails_helper'
+
+feature "users can add books" do
+
+  before(:each) do
+    Genre.create(genre_name: "Comedy")
+    Genre.create(genre_name: "Drama")
+    Genre.create(genre_name: "Non-Fiction")
+    Genre.create(genre_name: "Realistic Fiction")
+    Genre.create(genre_name: "Romance Novel")
+    Genre.create(genre_name: "Satire")
+    Genre.create(genre_name: "Tragedy")
+    Genre.create(genre_name: "TragiComedy")
+  end
+
+  let!(:author) { Author.new(name: "Matthew Lewis") }
+
+  let!(:book_1) {
+    FactoryGirl.create(
+    :book,
```

Comment:
Align the parameters of a method call if they span more than one line.<br>Indent the first parameter one step more than the start of the previous line.